packages/desktop-ui/tsconfig.lib.json

```diff
@@ -1,18 +1,18 @@
 {
-  "extends": "./tsconfig.json",
+  "extends": "../../node_modules/ng-packagr/lib/ts/conf/tsconfig.ngc.json",
   "compilerOptions": {
-    "declaration": true,
-    "declarationMap": true,
-    "inlineSources": true,
-    "types": []
+    "baseUrl": ".",
+    "outDir": "dist",
+    "declarationDir": "dist",
+    "paths": {
+      "@deepkit/core": ["../core/src/index.ts"],
+      "@deepkit/core-rxjs": ["../core-rxjs/src/index.ts"],
+      "@deepkit/event": ["../event/src/index.ts"],
+      "@deepkit/type": ["../type/src/index.ts"]
+    }
```

Comment:
`ng-packagr` doesn't work with TypeScript 5.2.2 and project references

Reply:
we don't use ng-packagr anyways

Reply:
@marcj we use ng-packagr for building the this package

Reply:
Oh I see, that was recently introduced. The library was designed to be TS only, so we could probably remove that again if it causes any issues